Doc/library/sys.rst

```diff
@@ -787,6 +787,22 @@ always available.
    .. versionadded:: 3.2
 
 
+.. function:: _getcallingmodule([depth])
```

Comment:
Minor bikeshedding.  `_getcallingmodule()` feels weird to me because modules don't often call things and `_getcallingmodule()` isn't returning the name of a module that called you.  

The alternative that comes to mind is `_getframemodule()` which evokes `_getframe()`. Since the two function's purposes are related, it seems like the names should be related too.

Reply:
I see your point, though enough callers are `<module>` that we couldn't just grab the value from the function object 😉 (e.g. defining dataclasses, named tuples, or the typing objects that need this).

`_getframemodulename()` is probably best, since it does only return the name (and mostly we want the name - only doctest uses it to get the module itself), but I'll wait for a second vote.

Reply:
`_getframemodulename()` works for me!